static/options.ts

```diff
@@ -36,4 +38,4 @@ declare var __webpack_public_path__: string;
 
 __webpack_public_path__ = window.staticRoot;
 
-export const options = window.compilerExplorerOptions;
+export const options: CompilerExplorerOptions & {[key: string]: unknown} = window.compilerExplorerOptions as CompilerExplorerOptions & {[key: string]: unknown} ;
```

Comment:
Surely there must be a better way to handle this until we fully move to TS, right?

Reply:
Could we try something like this?

```patch
diff --git a/static/global.ts b/static/global.ts
index 10703670..1ec1e828 100644
--- a/static/global.ts
+++ b/static/global.ts
@@ -22,7 +22,12 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-type CompilerExplorerOptions = Record<string, unknown>;
+import { Libs } from './options.interfaces';
+
+type CompilerExplorerOptions = Record<string, unknown> & {
+    libs: Libs;
+    defaultLibs: Record<string, string | null>
+};
 
 declare global {
     export interface Window {
diff --git a/static/options.ts b/static/options.ts
index 4ce5e075..d92ebb5a 100644
--- a/static/options.ts
+++ b/static/options.ts
@@ -22,8 +22,6 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-import { Options } from './options.interfaces';
-
 const configElement = document.getElementById('config');
 
 window.httpRoot = configElement.getAttribute('httpRoot');
@@ -38,4 +36,4 @@ declare var __webpack_public_path__: string;
 
 __webpack_public_path__ = window.staticRoot;
 
-export const options: Options & {[key: string]: unknown} = window.compilerExplorerOptions as Options & {[key: string]: unknown} ;
+export const options = window.compilerExplorerOptions
```

Reply:
Can you check if the what I changed works? I noticed the & right hand side was just options from Options

Reply:
Sorry for the delay, looks good to me :+1: